src/BecklynMonitoringBundle.php

```diff
@@ -28,15 +30,22 @@ public function __construct ()
     /**
      * @inheritDoc
      */
-    public function build (ContainerBuilder $container)
+    public function build (ContainerBuilder $container) : void
     {
-        $container->addCompilerPass(
-            new RegisterAssetNamespacesCompilerPass([
-                "monitoring" => __DIR__ . "/../build",
-            ])
-        );
-
-        $container->addCompilerPass($this->releaseVersionPass);
+        if (\class_exists(RegisterAssetNamespacesCompilerPass::class))
+        {
+            $container->addCompilerPass(
+                new RegisterAssetNamespacesCompilerPass([
+                    "monitoring" => __DIR__ . "/../src/Resources/public",
+                ])
+            );
+
+            $container->addCompilerPass($this->releaseVersionPass);
+        }
+        else if (!\class_exists(Packages::class))
+        {
+            throw new AssetIntegrationFailedException("No asset integration extension found. Please either install `becklyn/assets-bundle` or `symfony/asset` to use this bundle.");
```

Comment:
I would throw this exception when using the Twig function – because it is perfectly fine to use this bundle for Integration with Sentry only.